frb_codegen/Cargo.toml

```diff
@@ -1,12 +1,12 @@
 [package]
 name = "flutter_rust_bridge_codegen"
```

Comment:
Btw have you tried run `just release 1.xx.x 1.xx.x`? Maybe try to run it and ensure there is no error (except for the parts without permission)

(unrelated to this line of code)

Reply:
What do you mean exactly ? sorry I don't get it ^^'

Reply:
Run `just release the_previous_version the_next_version` on your computer and see whether it errors :)

Reply:
well the latest commit is from `just release` but locally it failed at:

```sh
# ...
git commit -m "bump from 1.48.0 to 1.49.0"
[chore/workspace ff28cf8c] bump from 1.48.0 to 1.49.0
 25 files changed, 39 insertions(+), 28 deletions(-)
 create mode 100644 .vscode/settings.json
git push
Enumerating objects: 87, done.
Counting objects: 100% (87/87), done.
Delta compression using up to 10 threads
Compressing objects: 100% (47/47), done.
Writing objects: 100% (48/48), 8.75 KiB | 4.37 MiB/s, done.
Total 48 (delta 40), reused 0 (delta 0), pack-reused 0
remote: Resolving deltas: 100% (40/40), completed with 34 local objects.
To https://github.com/Roms1383/flutter_rust_bridge.git
   10430e13..ff28cf8c  chore/workspace -> chore/workspace
awk '/## 1.49.0/{flag=1; next} /## 1.48.0/{flag=0} flag' CHANGELOG.md | gh release create v1.49.0 --notes-file "-" --draft --title v1.49.0
HTTP 404: Not Found (https://api.github.com/repos/fzyzcjy/flutter_rust_bridge/releases)
error: Recipe `release` failed on line 138 with exit code 1
```

Reply:
seems like it worked though

Reply:
let me revert the commit from `just release`, I think we're good @fzyzcjy.

Reply:
Great!